opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BaseApi.java

```diff
@@ -472,6 +472,17 @@ public void setStringProvider(StringProvider stringProvider) {
         this.stringProvider = stringProvider;
     }
 
+    protected String buildHttpGetParams(Map<String, String> params,
+                                        String encoding) throws UnsupportedEncodingException {
```

Comment:
I'm not a fan of having this logic twice, once for ``Map`` and once for ``List<NameValuePair>``. I prefer ``List<NameValuePair>``, as it more accurately maps to HTTP querystring parameters – and is the same type that the Apache HTTP client expects for POST request data.